### PR TITLE
Add thirdparty folders to sdist

### PR DESCRIPTION
**Pull request by mbkumar**
Thirdparty folders are now included into the source distribution uploaded to pypa.

### CODE REVIEW

File: MANIFEST.in [hunk 1]
```diff
@@ -7,5 +6,7 @@ recursive-exclude ci *
 recursive-exclude conda.recipe *
 recursive-exclude examples wout*.nc mercier* jxbout* spec*.sp spec*.sp.* .gitignore
 recursive-exclude tests .gitignore
+recursive-include thirdparty
+recursive-exclude thirdparty .git .gitignore .github/* .appveyor.yml *readthedocs.yml .travis.yml azure-pipelines.yml
```
Comment (florianwechsung) [inline MANIFEST.in:10]: I'm suprised that `thirdparty` is still among the folder behind `recursive-exclude`, is that correct?

Comment (mbkumar) [reply MANIFEST.in:10]: The command is `recursive-exclude dir pattern1 pattern2 ...`
All the files matching subsequent patterns on the same line should be excluded from the thirdparty folder. 
However, I am not sure about the interaction between lines 9 and 10.

**Comment by florianwechsung** [comment]
I'm not sure how to test this without merging and then creating a release. So let's do that and if it doesn't work we may have to make more changes.

File: MANIFEST.in (hunk 1 above)
Comment (mbkumar) [reply MANIFEST.in:10]: Just checked the [packages built](https://github.com/hiddenSymmetries/simsopt/suites/6210409075/artifacts/219589510). It is working as expected.

**Review by florianwechsung — APPROVED** [review]

**Comment by mbkumar** [comment]
> I'm not sure how to test this without merging and then creating a release. So let's do that and if it doesn't work we may have to make more changes.

You can download the wheels and sdist created in the [artifact] (https://github.com/hiddenSymmetries/simsopt/suites/6210409075/artifacts/219589510) from the wheel workflow.

**Pull request merged**